examples/lines_bars_and_markers/scatter_masked.py

```diff
@@ -3,30 +3,77 @@
 Scatter Masked
 ==============
 
-Mask some data points and add a line demarking
-masked regions.
+A NumPy masked array (see `numpy.ma`) can be passed to `.Axes.scatter` or
+`.pyplot.scatter` as the value of the *s* parameter in order to exclude certain
+data points from the plot.
+
+This example uses this technique so that data points within a particular radius
+are not included in the plot.
 
 """
 import matplotlib.pyplot as plt
 import numpy as np
 
-# Fixing random state for reproducibility
+# Fix random state for reproducibility
 np.random.seed(19680801)
 
-
+# Create N random (x, y) points
 N = 100
-r0 = 0.6
 x = 0.9 * np.random.rand(N)
 y = 0.9 * np.random.rand(N)
-area = (20 * np.random.rand(N))**2  # 0 to 10 point radii
-c = np.sqrt(area)
-r = np.sqrt(x ** 2 + y ** 2)
-area1 = np.ma.masked_where(r < r0, area)
-area2 = np.ma.masked_where(r >= r0, area)
-plt.scatter(x, y, s=area1, marker='^', c=c)
-plt.scatter(x, y, s=area2, marker='o', c=c)
-# Show the boundary between the regions:
-theta = np.arange(0, np.pi / 2, 0.01)
-plt.plot(r0 * np.cos(theta), r0 * np.sin(theta))
+
+# Create masked size array based on calculation of x and y values
+size = np.full(N, 36)
+radius = 0.6
+masked_size = np.ma.masked_where(radius > np.sqrt(x ** 2 + y ** 2), size)
+
+# Plot data points using masked array
+subplot_kw = {
+    'aspect': 'equal',
+    'xlim': (0, max(x)),
+    'ylim': (0, max(y))
+}
```

Comment:
It's a nice clean approach if you want to create two different plots with the same axes properties, rather than calling methods to set them 🙂 

Fair point, what if I moved the comment like this:

```python
subplot_kw = {
    'aspect': 'equal',
    'xlim': (0, max(x)),
    'ylim': (0, max(y))
}
fig, ax = plt.subplots(subplot_kw=subplot_kw)

# Plot data points using masked array
ax.scatter(x, y, s=masked_size, marker='^', c="mediumseagreen")
```